Starter.Api/Starter.Api.csproj

```diff
@@ -2,6 +2,8 @@
 
     <PropertyGroup>
         <TargetFramework>net6.0</TargetFramework>
+        <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
     </PropertyGroup>
 
     <ItemGroup>
```

Comment:
If we leave `NewtonsoftJson` package it should be also updated to the latest version